modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -94,3 +94,24 @@ Using `salt-thin` can be enabled as a fallback method, but it requires Python 3
 This method is not recommended nor supported and exists for development purposes only.
 Set [option]``web.ssh_use_salt_thin`` to `true` in the [path]``/etc/rhn/rhn.conf`` configuration file.
 ====
+
+== Salt Bundle functionality extension with PIP modules
+
+The Salt Bundle includes [literal]``PIP`` Python module to make it possible to extend the functionality of the bundled Salt Minion with extra Python modules.
+
+The default location to deploy modules on calling [command]``salt <minion_id> pip.install <module_name>`` is [path]``/var/lib/venv-salt-minion/local``.
+
+[NOTE]
+====
+In case if [path]``/var/lib/venv-salt-minion/local`` is not applicable for some reasons and the other location for the modules is required the destination path can be overridden by setting [literal]``VENV_PIP_TARGET`` environment variable for the [literal]``venv-salt-minion.service``.
+It is recommended to use systemd drop-in config file for the service. It could be done with the config file [path]``/etc/systemd/system/venv-salt-minion.service.d/10-pip-destination.conf``:
+----
+[Service]
+Environment=VENV_PIP_TARGET=/new/path/local/venv-salt-minion/pip
+----
+====
+
+[IMPORTANT]
+====
+The Salt Bundle does not maintain the modules installed with [literal]``PIP``. In case of updating the Salt Bundle it is possible that modules should be reinstalled, for example on updating the Salt Bundle with updated Python inside.
```

Comment:
```suggestion
The Salt Bundle does not keep Python packages installed through [literal]``pip`` on updates. To ensure that such packages are available after an update, it is recommended to install them with a Salt state that is applied after Salt Bundle updates.
```

Reply:
It is probably not needed to explain the implementation too detailed.  I'd basically say:

> The Salt Bundle is self-contained and works independent from installed system components such as the Python interpreter and the Python modules.
> The Salt Bundle does neither update nor uninstall a system component such as Python.

I'm not sure whether we must explicitly say that updating Python will happen in the run of regular system updates.

Reply:
The point here is that a user can install additional Python packages into the bundle, but those aren't updated when everything else in the bundle (Python, Salt, the Python packages we include) is updated. 

I previously thought that a Salt Bundle update would remove these additionally installed Python packages (which might still happen once we tackle https://confluence.suse.com/display/SUSEMANAGER/Problems+around+updating+Salt+Minion+%28bundled%29+with+Salt), but apparently that's not the case now. Either way, while these additional packages are placed into the bundle, it needs to be clear that the bundle is not actively doing anything with them.

Reply:
Is it sounds good if reword it this way?
```suggestion
The Python packages installed through [literal]``pip`` are not changing on updating the Salt Bundle. To ensure that such packages are available and functional after an update, it is recommended to install them with a Salt state that is applied after Salt Bundle updates.
```

Reply:
Sounds good!